elrond-wasm-debug/src/blockchain_mock.rs

```diff
@@ -144,7 +158,7 @@ impl BlockchainMock {
 	fn get_new_address(&self, creator_address: Address, creator_nonce: u64) -> Option<Address> {
 		self.new_addresses
 			.get(&(creator_address, creator_nonce))
-			.map(|addr_ref| addr_ref.clone())
+			.cloned()
```

Comment:
Typo or intentional? Shouldn't it be .clone() instead of .cloned()?

Reply:
.cloned() is the iterator version

Reply:
it means "clone each item from the iterator"

Reply:
clippy proposed this